app/src/main/java/com/vci/vectorcamapp/landing/presentation/LandingScreen.kt

```diff
@@ -104,7 +104,7 @@ fun LandingScreen(
             },
             text = {
                 Text(
-                    text = "You have an incomplete surveillance session. Resume where you left off?",
+                    text = "You have an in progress surveillance session. Resume where you left off?",
```

Comment:
Suggestion: "You have an surveillance session in progress. Resume where you left off?"